package.json

```diff
@@ -55,78 +55,73 @@
     "test": "npm run check:types && npm run lint:style && npm run verify verify:source",
     "test-ui": "vitest --ui",
     "coverage": "vitest run --coverage",
-    "test:storybook": "test-storybook --failOnConsole --browsers chromium --maxWorkers=1",
-    "test:storybook-ci": "\n      npx playwright install --with-deps chromium        && npx concurrently -k -s first -n \"SB,TEST\" -c \"magenta,blue\"          \"npm run storybook:build --quiet && npx http-server storybook-static --port 6006 --silent\"          \"npx wait-on tcp:6006 && npm run test:storybook\"\n    ",
+    "test:storybook": "npx vitest --projec=storybook",
```

Comment:
The `npx` can be dropped here. Also typo in `--projec`?

Reply:
done